format-check.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+set -ex
+
+for f in $(find vita3k tools/gen-modules tools/native-tool \( -name *.cpp -o -name *.h \)); do
```

Comment:
```suggestion
for f in $(git diff --name-only ${GITHUB_BASE_REF:-HEAD~1} HEAD)
```
You could try like this. Not fully sure if it works as intended

Reply:
Scratch that, also needs a check if it is `.cpp` or `.h` So add smth like
```bash
if [ "${f%.cpp}" != "${f}" ] || [ "${f%.h}" != "${f}" ]
``` 
around the other if block